native/src/components/Pagination.tsx

```diff
@@ -16,12 +16,12 @@ const DotsContainer = styled.View`
 `
 
 const Dot = styled(Pressable)<{ isActive: boolean }>`
-  width: 10px;
-  height: 10px;
+  width: 12px;
+  height: 12px;
   border-radius: 5px;
-  margin: 0 4px;
+  margin: 0 18px;
```

Comment:
:thought_balloon: Perhaps we should increase the touch area that responds to press, I find it quite hard at the moment to actually press a dot and go to a specific slide

Reply:
I will increase it more then